### PR TITLE
chore: add test for dock taskmanager rolebinemodel

### PR DESCRIPTION
**Pull request by tsic404**
log: as title

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -23,6 +23,7 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 option(BUILD_WITH_X11 "Build X11 emulation" ON)
+option(BUILD_UNINT_TEST "Build unint test" ON)
```
Comment (18202781743) [inline CMakeLists.txt:26]: 单元测试是unit_test吧，

File: tests/panels/dock/taskmanager/combinemodelb.cpp [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        case dataRole:
+            return m_list[index.row()]->data();
+    }
+
+    return {};
+}
+
+void TestModelB::addData(DataB *data)
+{
+    beginInsertRows(QModelIndex(), m_list.size(), m_list.size() + 1);
```
Comment (18202781743) [inline tests/panels/dock/taskmanager/combinemodelb.cpp:73]:
```suggestion
    beginInsertRows(QModelIndex(), m_list.size(), m_list.size());
```

Comment (18202781743) [reply tests/panels/dock/taskmanager/combinemodelb.cpp:73]: data没有释放，

File: CMakeLists.txt [hunk 3]
```diff
@@ -23,6 +23,7 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 option(BUILD_WITH_X11 "Build X11 emulation" ON)
+option(BUILD_UNIT_TEST "Build uint test" ON)
```
Comment (BLumia) [inline CMakeLists.txt:26]: 建议叫 `BUILD_TESTING`，和 ctest 的默认选项名称一致。

File: CMakeLists.txt [hunk 4]
```diff
@@ -60,6 +61,12 @@ add_subdirectory(example)
 add_subdirectory(panels)
 add_subdirectory(applets)
 
+if(BUILD_UNIT_TEST)
```
Comment (BLumia) [inline CMakeLists.txt:64]: `enable_testing` 的文档

> This command is automatically invoked when the [CTest](http://127.0.0.1:10715/cmake/cmake.org/cmake/help/v3.29/module/CTest.html#module:CTest) module is included, except if the BUILD_TESTING option is turned off.

**Review by BLumia — DISMISSED** [review]

**Pull request merged**